stingray/tests/test_spectroscopy.py

```diff
@@ -395,3 +391,47 @@ def test_compute_rms():
 
     with pytest.raises(ValueError):
         spec.compute_rms(cs, model, criteria="filter")
+
+
+def test_get_mean_phase_difference():
+    dt = 0.01
+    time = np.arange(0, 100, dt)
+    qpo_freq = 0.1
+    harmonic_freq = 2 * qpo_freq
+    counts_qpo = 100 * np.sin(2 * np.pi * qpo_freq * time - np.pi / 4)
+    counts_harmonic = 50 * np.sin(2 * np.pi * harmonic_freq * time - np.pi / 4)
+    counts = counts_qpo + counts_harmonic
+
+    lc = Lightcurve(time, counts, dt=dt)
+    cs = Crossspectrum(lc, lc)
+
+    model = models.Lorentz1D(x_0=qpo_freq, amplitude=100) + models.Lorentz1D(
+        x_0=harmonic_freq, amplitude=50
+    )
+    avg_psi, stddev = spec.get_mean_phase_difference(cs, model)
+
+    assert np.isclose(avg_psi, np.pi / 2, 1e-5)
+    assert np.isclose(stddev, np.pi / 2, 1e-5)
+
+
+def test_get_phase_lag():
+    dt = 0.01
+    time = np.arange(0, 100, dt)
+    qpo_freq = 0.1
+    harmonic_freq = 2 * qpo_freq
+    counts_qpo = 100 * np.sin(2 * np.pi * qpo_freq * time + np.pi / 4)
+    counts_harmonic = 50 * np.sin(2 * np.pi * harmonic_freq * time + np.pi / 4)
```

Comment:
Same comment as above - it looks like the phases here are the same. I think I am also misunderstanding something, but should cap_phi_1 and cap_phi_2 not be pi/4 then, and small_psi 0?